### PR TITLE
Limit the amount of runs returned by GetRunsActive.

### PR DESCRIPTION
**Pull request by dsuhinin**
What's inside the box:
- apply default limit - `50` for AIM `GetRunsActive` endpoint to decrease overall pressure to database, service and UI.

### CODE REVIEW

File: pkg/api/aim/runs.go [hunk 1]
```diff
@@ -240,6 +240,7 @@ func GetRunsActive(c *fiber.Ctx) error {
 	log.Debugf("getRunsActive namespace: %s", ns.Code)
 
 	q := struct {
+		Limit          int  `query:"limit"`
```
Comment (jgiannuzzi) [inline pkg/api/aim/runs.go:243]: This is not supported by the UI at this stage (and probably never will)

Comment (dsuhinin) [reply pkg/api/aim/runs.go:243]: correct. I though that we want to just limit default case.

Comment (jgiannuzzi) [reply pkg/api/aim/runs.go:243]: for now I would suggest to simply hard-code the limit to 100.

Comment (dsuhinin) [reply pkg/api/aim/runs.go:243]: ok.

Comment (dsuhinin) [reply pkg/api/aim/runs.go]: hardcoded.

**Review by jgiannuzzi — DISMISSED** [review]

**Review by jgiannuzzi — APPROVED** [review]

**Pull request merged**